test/test_transforms.py

```diff
@@ -149,3 +149,46 @@ def test_truncate(self):
     def test_truncate_jit(self):
         """test truncation with scripting on both sequence and batch of sequence with both str and int types"""
         self._truncate(test_scripting=True)
+
+
+class TestGPT2BPETokenizer(TorchtextTestCase):
+    def _gpt2_bpe_tokenizer(self, test_scripting):
+        encoder_json = "gpt2_bpe_encoder.json"
+        bpe_vocab = "gpt2_bpe_vocab.bpe"
+        tokenizer = transforms.GPT2BPETokenizer.get_gpt2_bpe_tokenizer(
+            encoder_json_path=get_asset_path(encoder_json),
+            vocab_bpe_path=get_asset_path(bpe_vocab),
```

Comment:
we should add them to pytorch/text aws folder as artifacts so that we could use them with pre-trained models.